CHANGELOG.md

```diff
@@ -4,9 +4,29 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v3.0.0](https://github.com/voxpupuli/puppet-varnish/tree/v3.0.0) (2023-02-28)
+## [v3.1.0](https://github.com/voxpupuli/puppet-varnish/tree/v3.1.0) (2023-05-12)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-varnish/compare/d55e143663f24b4f5efd8a9628a3d0173264609b...v3.0.0)
+[Full Changelog](https://github.com/voxpupuli/puppet-varnish/compare/v3.0.0...v3.1.0)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#20](https://github.com/voxpupuli/puppet-varnish/pull/20) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
the release needs to be a major one (4.0.0) because of this PR

Reply:
Thanks, did major instead of minor